### DISCUSSION TITLE
Card builder

### DISCUSSION THREAD

**Discussion by nathanchu** [anuraghazra/github-readme-stats#723] [Ideas]
I think this was mentioned in some issue I cannot find anymore. There should be a card "builder" to make choosing a card easier.

**Comment by nathanchu** [comment]
I was starting to work on this, but it would be easier to build if there was a json api specification for this that could be requested that would have all of the api information on it, such as:

```json
{
  "/api": {
    "params": [
      {
        "name": "username",
        "required": true,
        "default": null,
        "description": "Your username"
      }
    ]
  }
}
```
But obviously with all of the query params and routes. It could also be used to generate the readme if new PRs just added to the json file.

**Comment by rickstaa** [comment]
For future reference, @anuraghazra is working on implementing a URL builder with a live visual preview (see https://github.com/anuraghazra/github-readme-stats/issues/1355#issuecomment-933709440).
